cmd/manager/main.go

```diff
@@ -6,8 +6,8 @@ import (
 	"fmt"
 	"runtime"
 
-	"github.com/mumoshu/aws-secret-operator/pkg/apis"
-	"github.com/mumoshu/aws-secret-operator/pkg/controller"
+	"aws-secret-operator/pkg/apis"
```

Comment:
I thought the local imports work only outside of the GOPATH, so does this mean you're building aws-secret-operator somewhere outside of GOPATH, right?